examples/Fine_tuning_for_function_calling.ipynb

```diff
@@ -75,14 +75,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# !pip install tenacity\n",
-    "# !pip install openai\n",
-    "# !pip install typing"
+    "!pip install tenacity -q\n",
+    "!pip install openai -q\n",
+    "!pip install typing -q"
```

Comment:
nit/tip: you can remove the `-q` and instead just remove the cell output before committing!